api-operator/deploy/controller-artifacts/operator-pem-secret.yaml

```diff
@@ -0,0 +1,24 @@
+#Copyright (c)  WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Config map containing the certificate for operator rest server
+
+apiVersion: v1
+kind: Secret
+metadata:
+  name: operator-pem-secret
+data:
+  operator.pem: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURxVENDQXBHZ0F3SUJBZ0lVZklXNzdzdWY3QWlFelorYlJKazF2WjhsM2Jrd0RRWUpLb1pJaHZjTkFRRUwKQlFBd1pERUxNQWtHQTFVRUJoTUNWVk14Q3pBSkJnTlZCQWdNQWtOQk1SWXdGQVlEVlFRSERBMU5iM1Z1ZEdGcApiaUJXYVdWM01RMHdDd1lEVlFRS0RBUlhVMDh5TVEwd0N3WURWUVFMREFSWFUwOHlNUkl3RUFZRFZRUUREQWxzCmIyTmhiR2h2YzNRd0hoY05NakF4TWpBNU1EUXdNVEkyV2hjTk16QXhNakEzTURRd01USTJXakJrTVFzd0NRWUQKVlFRR0V3SlZVekVMTUFrR0ExVUVDQXdDUTBFeEZqQVVCZ05WQkFjTURVMXZkVzUwWVdsdUlGWnBaWGN4RFRBTApCZ05WQkFvTUJGZFRUekl4RFRBTEJnTlZCQXNNQkZkVFR6SXhFakFRQmdOVkJBTU1DV3h2WTJGc2FHOXpkRENDCkFTSXdEUVlKS29aSWh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBTEhDWkRhdlpuV1BiWno0cGRBbDVyOGMKcjBwbWdUSnBId3Q4a0dMWnVRN1FmR2Z1anRNKytMZ2U2bVA2R2l4UFJnWTRCaW5QZWFEV2VBS0pJMUE2Z0hydwpycHI1aDJQNkZ6RDBQdStTa2hSQWxydlJoUmtKQ0VYQnJYMTVRaStUbVpHNTZIR1NmblVwOTAvb29qU3QzZU1YCkpOVE96S2hRLzY1U0Fnb1dBMHMvN1FIZmNGYzZ3OUxqc3A2cjFRb2FVc2h4VjNzOFlhRjdFWnVnYzF6dHNCWE8KTHlPNFJkSDJHVXRwNDlNeHhtcDRZWjJ0eUpicVZnQzZ5N2UwOGpCOCsxYkF0U3Flc3VENklkNDg5ZStGSm1ITApUdldvMEhPNFBTZHE4TDJpdW45SUZtRTZqenVXOFlSVEl3V3hNR0V5THZkWGsrdmZnVzhTK2tOT2Z3SFVUWmtDCkF3RUFBYU5UTUZFd0hRWURWUjBPQkJZRUZEYmY3QTdIVjllU1VNS2lkUEdIdnlKVzZocnpNQjhHQTFVZEl3UVkKTUJhQUZEYmY3QTdIVjllU1VNS2lkUEdIdnlKVzZocnpNQThHQTFVZEV3RUIvd1FGTUFNQkFmOHdEUVlKS29aSQpodmNOQVFFTEJRQURnZ0VCQUF1VGxTaFdPeDdiTjFKaHBzNndSZ1U2TThjdm1lWitMTGtTMkF3Wjc1c0tOcFQwCnF4WGloM1VrcUFaQ09JRXM5d0NaS1JuUkp1QlNPSzU1aDR4RjRLTnFleFRGa2JneE9NbmNTK014ZzY5czJuSWcKcm44ZkVIcldmdHVhM3Zzbk1kTmZrM0FqQ1hmbFJKYjB4NUo5dkRJUTdxeFNWVGVIZTRnTGFhMnV6elV2Y1hxTQpIMkJQS25WTUdpM2VwWEtIUnhvanlZTHVUbWZlRkJYQW5iRkJNK3NCREdnMWs5cUtDM0I2amxPVkQzZlhQRXFCCk5sUFBralJHSTlwSTgyTnBFSWU5VFdkN080MHNkMk02ZE1oYjhleVR0b2huRXBaRFoxSWRDSTkwNHJWVVNFaXAKalUwTnFHOXpnRXJiTEY3TGhydGFJMllHQkJwNUxWbFJ4Q3RHZVJrPQotLS0tLUVORCBDRVJUSUZJQ0FURS0tLS0t
```

Comment:
WIthout having 2 secrets shall we use the secret relevant to the following kubectl command?

kubectl create secret tls ${CERT_NAME} --key ${KEY_FILE} --cert ${CERT_FILE}

Reply:
Fixed with 25bb451d45419839cbeaebd100ce6d4d66e74413